<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QDialog" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="findGameButton">
   <property name="geometry">
    <rect>
     <x>729</x>
     <y>490</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск игры</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>511</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="BoxMode">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string>Режим игры</string>
      </property>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>54</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="modeTeam">
          <property name="text">
           <string>Командный</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="modeSolo">
          <property name="text">
           <string>Одиночный</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="BoxLevel">
      <property name="title">
       <string>Уровень игры</string>
      </property>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>95</width>
         <height>83</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="levelEasy">
          <property name="text">
           <string>Легкий</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="levelMedium">
          <property name="text">
           <string>Средний</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="levelHard">
          <property name="text">
           <string>Сложный</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="BoxLength">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Длина раунда (сек)</string>
      </property>
      <widget class="QRadioButton" name="length60">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>60</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="length90">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>90</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="length120">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>112</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>120</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
